net-misc/networkmanager-strongswan/networkmanager-strongswan-1.4.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P=${P/networkmanager/NetworkManager}
+
+DESCRIPTION="NetworkManager StrongSwan plugin"
+HOMEPAGE="https://www.strongswan.org/"
+SRC_URI="https://download.strongswan.org/NetworkManager/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="glib"
+
+RDEPEND="net-misc/networkmanager
```

Comment:
```
checking for LIBNMA... no
configure: error: Package requirements (libnma >= 1.1.0) were not met:

No package 'libnma' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBNMA_CFLAGS
and LIBNMA_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

!!! Please attach the following file when seeking support:
!!! /tmp/portage/net-misc/networkmanager-strongswan-1.4.4/work/NetworkManager-strongswan-1.4.4/config.log
```

Missing USE-dep?

Reply:
Whoops. Added them, please test again.